### PR TITLE
Implemented circleci

### PR DESCRIPTION
**Pull request by EfazAhmed**

### CODE REVIEW

File: circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    # Be sure to update the Docker image tag below to the openjdk version of your application.
+    # A list of available CircleCI Docker Convenience assets are available here: https://circleci.com/developer/assets/image/cimg/openjdk
+    docker:
+      - image: cimg/openjdk:8.0
+      # Add steps to the job
+      # See: https://circleci.com/docs/2.0/configuration-reference/#steps
+    steps:
+      # Check out the code as the first step.
+      - checkout:
+          path: ~/CiCdDemo
```
Comment (ragogoe24) [inline circleci/config.yml]: path: ~/Final-Project

**Review by ragogoe24 — CHANGES_REQUESTED** [review]

**Comment by Boschio** [comment]
LGTM

**Review by Boschio — APPROVED** [review]
LGTM

**Review by Boschio — APPROVED** [review]
LGTM

**Pull request merged**